plugins/source/facebookmarketing/rest/ad.go

```diff
@@ -0,0 +1,104 @@
+package rest
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+	"net/url"
+	"strings"
+)
+
+type Ad struct {
+	AccountId                         string         `json:"account_id"`
```

Comment:
`Id` -> `ID` per Go recommendations (here + resolvers)

Reply:
I really dislike this golang recommendation (because it makes it  hard for me to parse an object-name into 'words').
Also, this was autogenerated by a script from a JSON file that is 'snake-case', and the script really has no way of knowing it needs to change 'Id' to 'ID' (without adding some sort of 'english dictionary' capability to the python script, which I don't want to do..).